core/settings.gradle.kts

```diff
@@ -0,0 +1,9 @@
+// Configure the root project and subproject settings
+rootProject.name = "core"
+
+include(":ballerina")
+include(":native")
+
+// Rename projects
+project(":ballerina").name = "core-ballerina"
+project(":native").name = "core-native"
```

Comment:
should we rename to dicom-core-native & dicom-core-ballerina?

Reply:
As discussed, renamed to `dicom-ballerina` and `dicom-native` in https://github.com/ballerina-platform/module-ballerinax-health.dicom/commit/3807383514d097fe372c936aa46ef84a627a58d3